marketplace/netlify-build/README.md

```diff
@@ -0,0 +1,25 @@
+# Netlify Build and Preview
+
+This sidebar extension allows you to build and preview up to 3 Netlify sites.
+
+This is an alternative approach to using a webhook to trigger a build. The advantage
+is that the editor gets control on when a build is executed instead of responding
+to automatic events such as `Entry.publish`.
+
+![Netlify Sidebar UI Sample](./assets/netlify-triggering.gif)
+
+## Requirements
++ A Netlify site
++ Netlify webhooks setup, [see their documentation](https://www.netlify.com/docs/webhooks/)
+
+## Installation
+
+Navigate to the Extensions page to install the extension and provide the
+correct credentials into the required fields then click "Save".
```

Comment:
"correct configuration: build hook URL, site ID, URL and its name. To provide multiple configurations separate values with a comma."

Reply:
Added in more detail.